civictechprojects/static/css/_vars.scss

```diff
@@ -20,36 +20,104 @@ $color-disabled-border: #a8a8a8; // for e.g. a disabled state button border
 $color-grey-frame-border: #6b6e70; // from design guide
 $color-grey-unselected: #585858; // unselected item color
 
-// temporary holding zone for some DS colors
-//first, declare color rgb values to color names
-$color-green-10: #e6f7ef;
-$color-green-80: #018047;
-$color-blue-10: #e6effa;
-$color-neutral-50: #bdbdbd;
+// ##############################
+// Design System 
+// ##############################
+
+// # begin color scales
+
+// primary colors //
+
+//brand
+$color-brand-10: #fef5e6;
+$color-brand-20: #fde2b3;
+$color-brand-45: #fac567;
+$color-brand-60: #f9b135;
+$color-brand-70: #f79e02;
+
+//typography
+$color-neutral-80: #535a5f;
 $color-neutral-90: #222629;
 $color-neutral-100: #191919;
+$color-brand-80: #f06d23;
+
+//background
+$color-neutral-10: #ffffff;
+$color-neutral-15: #f6f7f8;
+$color-neutral-20: #f5f6f6;
+$color-neutral-30: #eaeef0;
+$color-neutral-45: #8c8c8c; // rgba(25,25,25,0.5) to rgb; may not be correct, waiting for design response 04/2022
+
+// border
+$color-neutral-35: #d4d4d4;
```

Comment:
"$color-disabled-border: #a8a8a8; // for e.g. a disabled state button border"
Can you please mark that $color-neutral-35: #d4d4d4 will be for any button/UI disabled border? I checked the updated disabled button, and it is not visible when the background is not #FFFFFF. ;( I will add a disabled border to each disabled button in Figma if we agree on it.

Reply:
Yep, makes sense. I'll add the comment and add the border color to disabled buttons.